test/integ_test/evaluator/docker_eval/dockerfile/Dockerfile

```diff
@@ -5,9 +5,9 @@
 FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04
 
 RUN apt-get -y update && apt-get -y upgrade
+RUN pip install torch torchvision --index-url https://download.pytorch.org/whl/cpu --no-cache-dir
```

Comment:
Is this line used to fix the error of `no space left`?

Reply:
yes, it is to avoid installing cuda stuff to save disk space